modules/web/js/ballerina/diagram2/views/default/components/nodes/top-level-nodes.jsx

```diff
@@ -199,8 +215,18 @@ class PackageDeclarationNode extends React.Component {
      */
     render() {
         const model = this.props.model;
-        const bBox = model.viewState.bBox;
+        // TODO fix for the model not being parsed properly
+        if (model.viewState.components.topLevelNodes === undefined) {
+            return <g />;
+        }
+        const bBox = model.viewState.components.topLevelNodes;
         const headerHeight = packageDefinition.header.height;
+
+        // TODO fix for the model not being parsed properly
+        if (this.props.model.viewState.components.importDeclaration === undefined ||
+            this.props.model.viewState.components.importsExpanded === undefined) {
+            return <g />;
```

Comment:
If you want to return an empty value, you can use
`return (null);`

Reply:
Fixed with commit 03ae4b2